### ISSUE TITLE
[All Versions] Open GL Error

### ISSUE THREAD

**Issue by justinrusso** [MJRLegends/ExtraPlanets#206]
Hey MJR,

With the help of TheBetweenland's GL debugger, I believe we've found ExtraPlanets to be the cause of our Open GL issues.

The log is quite large, but heres an excerpt I believe points to the mod:

```
[18:02:53] [main/INFO] [STDERR/]: [thebetweenlands.core.OpenGLDebug:checkThrowError:98]: thebetweenlands.core.OpenGLDebug$OpenGLError: An OpenGL Error has occurred: Invalid enum (1280)
[18:02:53] [main/INFO] [STDERR/]: [thebetweenlands.core.OpenGLDebug:checkThrowError:98]: 	at thebetweenlands.core.OpenGLDebug.checkThrowError(OpenGLDebug.java:98)
[18:02:53] [main/INFO] [STDERR/]: [thebetweenlands.core.OpenGLDebug:checkThrowError:98]: 	at com.mjr.extraplanets.client.handlers.MainHandlerClient.onRenderFogDensity(MainHandlerClient.java:287)

```

Log:  https://gist.github.com/justinrusso/2c6627cedb13fb1f97cb9ae8d527542e

**Comment by MJRLegends** [comment]
Thank for the report, yeah i have already fixed this issue, it will be fixed in the next update

**Comment by MJRLegends** [comment]
Updates for MC 1.8.9, MC 1.10.2, MC 1.11.2, MC 1.12.2 are live with this fixed, only MC 1.7.10 left with this issue atm

**Comment by MJRLegends** [comment]
Fixed in all versions
